### PR TITLE
Additional tests

### PR DESCRIPTION
**Pull request by zduey**
A couple of small changes and a few additional tests added

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -139,3 +139,11 @@ Generating reports on the command line::
                             contacts to exclude from classifier
       --joint-report, -j    include CatFR/FR for FR reports (default: off)
 
+Testing
```
Comment (mivade) [inline README.rst]: After we roll this out, let's setup a cronjob on RAM_clinical (or whatever account) to automate running these tests periodically.

File: ramutils/log.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def format_all(self, flush=True):
         if len(lines):
             return "ACCUMULATED WARNINGS\n--------------------\n" + '\n'.join(lines)
         else:
-            return None
+            return ''
```
Comment (mivade) [inline ramutils/log.py]: I think this probably fixes #90.

Comment (zduey) [reply ramutils/log.py]: Yes, that's correct.

File: README.rst (hunk 1 above)
Comment (zduey) [reply README.rst]: Yeah, I think that makes a lot of sense.

**Pull request merged**